### ISSUE TITLE
Homepage bluebox accessibility

### ISSUE THREAD

**Issue by dlowenberg** [CDL-Dryad/dryad-product-roadmap#592]
Under the hero image, the search form in the blue rectangle is missing a form label. Add a form field label. For visual consistency, treat this search box in the same way as you treat the search box in the top right-hand corner: 
Display "Search" in the search box
Have the "Search" text serve as both the placeholder text and the form field label 
Use a magnifying glass for the submit button
